lib/tasks/reminder.rake

```diff
@@ -0,0 +1,24 @@
+namespace :reminder do
+  desc "Send actionable approver reminder email for pending requests"
+  task actionable_approvers: :environment do
+    proposals = Proposal.pending.where(client_data_type: "Ncr::WorkOrder")
+    proposals.each do |proposal|
+      send_reminder_email(proposal)
+    end
+  end
+
+  private
+
+  def send_reminder_email(proposal)
+    user = proposal.client_data.current_approver
+    unless user
```

Comment:
for the record, I do not prefer `unless` to if! I love `if` and have some sort of weird mental block when it come to `unless`. Like, right now, I am thinking really hard about whether or not we will `return` here if there is a user.